packages/backend/src/plugins/graphql.ts

```diff
@@ -8,11 +9,12 @@ import { PluginEnvironment } from '../types';
 
 export default async function createPlugin({
   logger,
-  catalogClient,
+  discovery,
 }: PluginEnvironment): Promise<Router> {
+  const catalogClient = new CatalogClient({ discoveryApi: discovery });
   return await createRouter({
     modules: [Catalog()],
     logger,
     loaders: { ...createCatalogLoader(catalogClient) },
   });
-}
+}
```

Comment:
it's nice to turn on automatic line endings in the editor :)

Reply:
Good call, got that setup now and also updated this file 👍